kpi/permissions.py

```diff
@@ -273,3 +274,18 @@ class SubmissionValidationStatusPermission(SubmissionPermission):
     }
 
 
+class ReportPermission(IsOwnerOrReadOnly):
+    def has_object_permission(self, request, view, obj):
+        # Checks if the user has the require permissions
+        # To access the submission data in reports
+        user = request.user
+        if user.is_anonymous:
+            user = get_anonymous_user()
+        permissions = list(obj.get_perms(user))
```

Comment:
Let's add a validation if user is a super user and return `True`

```python
        if user.is_superuser:
            return True
            
        if user.is_anonymous:
            user = get_anonymous_user()
            
        permissions = list(obj.get_perms(user))
        
```

Reply:
Added the superuser to the permissions